system/systemd.py

```diff
@@ -225,13 +232,16 @@ def main():
                 enabled = dict(type='bool'),
                 masked = dict(type='bool'),
                 daemon_reload= dict(type='bool', default=False, aliases=['daemon-reload']),
+                user= dict(type='bool', default=False),
             ),
             supports_check_mode=True,
             required_one_of=[['state', 'enabled', 'masked', 'daemon_reload']],
         )
 
     # initialize
     systemctl = module.get_bin_path('systemctl')
+    if module.params['user']:
+        systemctl = systemctl + " --user"
```

Comment:
so i do get diff error messages depending on version of systemd, only the 'older ones' reference X and DISPLAY, testing mostly on CentOS which does not seem to have pam_systemd package available. Since this does not break existing functionality and might work if the system is 'correctly setup' I am adding it.